tasks/hyrax-metadata-updates/index.js

```diff
@@ -0,0 +1,290 @@
+'use strict';
+
+const cumulusMessageAdapter = require('@cumulus/cumulus-message-adapter-js');
+const { InvalidArgument } = require('@cumulus/errors');
+
+const get = require('lodash.get');
+const _ = require('lodash/core');
+const cloneDeep = require('lodash.clonedeep');
+
+const {
+  CMR
+} = require('@cumulus/cmr-client');
+
+const { isECHO10File, isUMMGFile, isCMRFilename } = require('@cumulus/cmrjs/cmr-utils');
```

Comment:
`isCMRFilename` seems to not be used.

Reply:
It's supposed to be. Looks like my updates to use it instead of f.type === 'metadata' got eclipsed. Will update.